src/main/java/edu/ucsb/ucsbcslas/repositories/TutorAssignmentRepository.java

```diff
@@ -11,7 +11,30 @@
 
 @Repository
 public interface TutorAssignmentRepository extends CrudRepository<TutorAssignment, Long> {
-  List<TutorAssignment> findAll();
-  List<TutorAssignment> findAllByCourse(Course course);
-  List<TutorAssignment> findAllByTutor(Tutor tutor);
-}
+    List<TutorAssignment> findAll();
+
+    List<TutorAssignment> findAllByCourse(Course course);
+
+    List<TutorAssignment> findAllByTutor(Tutor tutor);
+
+    List<TutorAssignment> findAllByCourseId(Long courseId);}
+
+    
+    
+        
+         
+
+        
+
```

Comment:
Similarly, it would be helpful to remove a chunk of empty lines like this one, a few are fine but it is better for readability if users don't have to scroll too far to see the next piece.